Bottom: a84b0f9f1e1ce603277a6e8b4ebbdfcb1f1c09d5
Top:    94b2bc6b721bc3fc8913576f08588412f57ee57a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-13 02:06:30 +0000

Refresh of add-routines-to-render-just-a-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 80b9203..bccc246 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -613,7 +613,7 @@ fill_polyarea (POLYAREA *pa, const BoxType *clip_box, double scale)
 
   /* Special case non-holed polygons which don't require a stencil bit */
   if (pa->contour_tree->size == 1) {
-    fill_contour (poly->Clipped->contours, scale);
+    fill_contour (pa->contours, scale);
     return;
   }
 
@@ -658,7 +658,7 @@ fill_polyarea (POLYAREA *pa, const BoxType *clip_box, double scale)
   /* Drawing operations as masked to areas where the stencil buffer is '0' */
 
   /* Draw the polygon outer */
-  fill_contour (poly->Clipped->contours, scale);
+  fill_contour (pa->contours, scale);
   hidgl_flush_triangles (&buffer);
 
   /* Unassign our stencil buffer bit */
